### ISSUE TITLE
[BUG] unregistered predefined rules are ignored

### ISSUE THREAD

**Issue by timostamm** [bufbuild/protovalidate-es#105]
## Description

When a schema uses predefined rules, protovalidate-es does not validate them by default. The extension must be passed to `createValidator()` in a registry.

This is problematic because the requirement to register the extension is not obvious. It is far too easy for a predefined rule to be added to a schema, only to be ignored by protovalidate-es.

## Steps to Reproduce

In the following example, we create a predefined rule and apply it to a field. Even though the message violates the rule, validation does not report any problems.

rules.proto:

```proto
syntax = "proto2";
package test.v1;

import "buf/validate/validate.proto";

extend buf.validate.StringRules {
  optional bool name = 81048952 [(buf.validate.predefined).cel = {
    id: "string.name"
    message: "value must be a valid name, a non-empty string with 50 or fewer characters"
    expression: "this.size() > 0 && this.size() <= 50"
  }];
}
```

person.proto

```proto
syntax = "proto3";
package test.v1;

import "buf/validate/validate.proto";
import "test/v1/rules.proto";

message Person {
  string first_name = 1 [(buf.validate.field).string.(name) = true];
  optional string middle_name = 2 [(buf.validate.field).string.(name) = true];
  string last_name = 3 [(buf.validate.field).string.(name) = true];
  optional string title = 4 [(buf.validate.field).string.(name) = true];
}
```

test.ts

```ts
import { createValidator, type ValidationResult } from "@bufbuild/protovalidate";
import { Person, PersonSchema } from "./gen/test/v1/person_pb.js";
import { create } from "@bufbuild/protobuf";

const validator = createValidator();

const person: Person = create(PersonSchema, {
  firstName: "John",
  lastName: "",
});

const result = validator.validate(PersonSchema, john);
console.log(result.kind); // "valid", but the field last_name is violating the predefined rule `test.name`.
```

## Expected Behavior

The validator should raise an error if it encounters an unknown predefined rule.

## Actual Behavior

The validator silently passes.

## Additional Context

https://github.com/bufbuild/protovalidate-es/issues/88
